### PR TITLE
Update to Bot API 6.8

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: site/docs/README.md [hunk 1]
```diff
@@ -100,8 +100,7 @@ Works! :tada:
 
 <div style="font-size: 0.75rem; display: flex; justify-content: center;">
 
-© 2021-2023 &middot; grammY supports Telegram Bot API 6.7 which was [released](https://core.telegram.org/bots/api#april-21-2023) on April 21, 2023.
-(Last highlight: multiple bot names, custom emoji, and better inline queries)
+© 2021-2023 &middot; grammY supports Telegram Bot API 6.8 which was [released](https://core.telegram.org/bots/api#april-21-2023) on August 18, 2023.
```
Comment (KnorpelSenf) [inline site/docs/README.md:103]:
```suggestion
© 2021-2023 &middot; grammY supports Telegram Bot API 6.8 which was [released](https://core.telegram.org/bots/api#april-21-2023) on August 18, 2023.
(Last highlight: Stories)
```

Comment (roj1512) [reply site/docs/README.md:103]: I thought it was nothing other than an empty object, but OK.

**Comment by CikiMomogi** [comment]
The link is not correct. https://core.telegram.org/bots/api#august-18-2023

File: site/docs/README.md [hunk 2]
```diff
@@ -100,8 +100,8 @@ Works! :tada:
...
```
Comment (CikiMomogi) [inline site/docs/README.md:103]:
```suggestion
© 2021-2023 &middot; grammY supports Telegram Bot API 6.8 which was [released](https://core.telegram.org/bots/api#august-18-2023) on August 18, 2023.
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]

**Review by CikiMomogi — APPROVED** [review]

**Review by KnorpelSenf — APPROVED** [review]

**Pull request merged**